sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/_models.py

```diff
@@ -0,0 +1,69 @@
+# ------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# -------------------------------------------------------------------------
+from typing import overload, List, Optional, Callable
+from azure.keyvault.secrets import SecretClient
+
+class AzureAppConfigurationKeyVaultOptions:
```

Comment:
In the `load_provider` docstring we just refer to this object as `KeyVaultOptions` (which I generally prefer).
Is there a particular reason for the more verbose name here?

Reply:
I guess the logic of the name is; 1 this is owned by App Config, not key vault. 2 we use this name in our other providers so it is nice that they match.

Reply:
We can leave it for now and wait till the archboard review.